cfpb/owning-a-home
# Feedback form

Add mini feedback form to OAH subpages

## Additions

- Feedback form html and css
- Preliminary js to show confirmation message & send params to feedback page

## Testing

- Check out branch & run frontendbuild.sh
- Verify that:
   - feedback module displays correctly on subpages 
  - Confirmation message appears after radio button is clicked & radio buttons are disabled
  - `is_helpful` choice is passed to feedback page when link is clicked

## Review

- @higs4281 
- @sonnakim 

## Screenshots

Explore rates:
<img width="1123" alt="screen shot 2016-11-18 at 10 13 19 am" src="https://cloud.githubusercontent.com/assets/778171/20441329/e15193f8-ad78-11e6-8080-eef98599a667.png">


Journey:

<img width="1125" alt="screen shot 2016-11-18 at 10 13 38 am" src="https://cloud.githubusercontent.com/assets/778171/20441324/dd138878-ad78-11e6-9912-49c58eb594f7.png">

Loan options:

<img width="1234" alt="screen shot 2016-11-18 at 10 13 56 am" src="https://cloud.githubusercontent.com/assets/778171/20441323/da26da02-ad78-11e6-902e-2e6b91d508f9.png">


## Todos

- `feedback.js` needs to be modularized; this is just a preliminary version to test functionality on build

## Checklist
